ui/src/main/resources/FileManager/Translations.pt_BR.xml

```diff
@@ -0,0 +1,35 @@
+<xwikidoc>
+  <web>FileManager</web>
+  <name>Translation</name>
+  <language>fr</language>
+  <defaultLanguage/>
```

Comment:
Looks like you edited the French translation. The language and default language are not good.